Source/driver/Castro_advance_ctu.cpp

```diff
@@ -147,14 +134,10 @@ Castro::do_advance_ctu(Real time,
 #endif                
                       old_source, Sborder, S_new, prev_time, dt, apply_sources_to_state);
 
-      // Apply the old sources to the sources for the hydro.
-      // Note that we are doing an add here, not a copy,
-      // in case we have already started with some source
-      // terms (e.g. the source term predictor, or the SDC source).
+      if (do_hydro) {
```

Comment:
Can you add a comment here to explain that this operation is just filling the ghost zones of Source_Type?

Reply:
done